.github/workflows/check-invariant-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Strict NoThunk test
+on:
```

Comment:
Before this is merged, a [`schedule` trigger](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule) should be added here, right?

Reply:
Yes!